src/elements/content-sidebar/ContentSidebar.js

```diff
@@ -107,17 +113,23 @@ class ContentSidebar extends React.Component<Props, State> {
             apiHost,
             cache,
             clientName,
+            language,
             requestInterceptor,
             responseInterceptor,
             sharedLink,
             sharedLinkPassword,
             token,
         } = props;
 
+        const commonHeaders = {
+            [HEADER_ACCEPT_LANGUAGE]: language || DEFAULT_LOCALE,
+        };
+
         this.api = new API({
             apiHost,
             cache,
             clientName,
+            commonHeaders,
```

Comment:
Can we pass in `language` and allow the API class to manage the headers? Otherwise each top-level Element will need to redefine the headers.

Reply:
In this case, that's fine by me. If a need to add more headers occurs, we'll want to switch back to `headers`